docs/source/stream.mdx

```diff
@@ -137,7 +137,7 @@ When you need to remove one or more columns, give [`IterableDataset.remove_colum
 
 ### Cast
 
-[`IterableDataset.cast`] changes the feature type of one or more columns. This method takes your new `Features` as its argument. The following sample code shows how to change the feature types of `ClassLabel` and `Value`:
+[`IterableDataset.cast_column`] changes the feature type of one or more columns. This method takes your new `Features` as its argument. The following sample code shows how to change the feature types of `ClassLabel` and `Value`:
```

Comment:
This part was correct: it explains `cast`, not `cast_column`

```suggestion
[`IterableDataset.cast`] changes the feature type of one or more columns. This method takes your new `Features` as its argument. The following sample code shows how to change the feature types of `ClassLabel` and `Value`:
```

Reply:
Ah, I see where I got confused! 😵‍💫  

`IterableDataset.cast` and `IterableDataset.rename_column` aren't shown in the current API reference because they aren't added [here](https://github.com/huggingface/datasets/blob/master/docs/source/package_reference/main_classes.mdx). I'll add these functions to `main_classes.mdx` and then they should be displayed properly (unless you left them out on purpose?)!